wikipedia/README.md

```diff
@@ -108,6 +109,7 @@ This track accepts the following parameters with Rally 0.8.0+ using `--track-par
 - Ingest Operations:
   - `ingest_bulk_size` (default: 100)
   - `as_ingest_clients` (default: [1,2,4,8,16])
+  - `as_target_throughputs` (default: [-1,-1,-1,-1,-1])
```

Comment:
Can you also add that the document per second will be `ingest_bulk_size * as_target_throughputs `?

Reply:
Good point, added a relevant note to each challenge section.